location/tests/tests_location.py

```diff
@@ -0,0 +1,102 @@
+from django.urls import reverse
+
+from rest_framework import status
+
+from company.models import Address
+from location.models import Location
+
+
+class TestValidateCompany:
+
+    def test_passwords_incorrect(self, custom_company, randomizer):
+        data = randomizer.location_data()
+        data.setdefault('password', custom_company.user_password)
+        data.setdefault('company', custom_company.id+10)
+        response = custom_company.user.post(reverse('location_new-list'), data=data, format='json')
+        assert response.status_code == status.HTTP_400_BAD_REQUEST
+        assert response.json()
+        assert response.content == b'{"company":["Invalid pk \\"11\\" - object does not exist."]}'
+        assert response.json()['company'] == ['Invalid pk "11" - object does not exist.']
```

Comment:
there is no need to check bytes if you check json

Reply:
done